index.html

```diff
@@ -1,22 +1,87 @@
 <!DOCTYPE html>
-<html lang="en">
+<body lang="en">
   <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1, shrink-to-fit=no"
-    />
-    <title>My Blog</title>
-    <link
-      href="//fonts.googleapis.com/css?family=Roboto:400,500,300"
-      rel="stylesheet"
-      type="text/css"
-    />
-    <link rel="stylesheet" href="css/style.css" />
-    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Pam's Blog</title>
+  <link rel="stylesheet" href="/css/style.css">
+  <link rel="shortcut icon" type="image/x-icon" href="img/Heart foods .jpeg"/>
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-  </body>
-</html>
+  <header>
+    <h1>Wellness life</h1>
+    <ul>
+      <li><a href="#"> Wellness Sports</a></li>
+      <li><a href="#"> Wellness Food</a></li>
+      <li><a href="#"> Wellness Life</a></li>
+    </ul>
+  </header>
+<article>
+  <img src="/img/wellness.jpeg" alt="Wellness page logo" class="logo"/>
+  <span>This Wellness Life, we are passionate about helping you live your best life possible. 
+    We practice a holistic approach and focus on key pillars of health and wellbeing including
+    mindset, movement, nourishment, rest and recovery. We provide coaching, education, mentoring,
+    resources and tools to help you keep stress at bay, move your body every day, eat to thrive, 
+    get restful sleep and live your most awesome, purpose-filled life.<br>
+    <a href="https://www.thiswellnesslife.com/">Wellness for Life</a></span>
+</article>
+<hr>
+<article class="section">
+<div class="sport">
+  <h2>Wellness Sports</h2>
+  <span>Sports have a blend of values that reflects on our day-to-day life. They provide a platform
```

Comment:
Generally we want to use p tags for paragraph text like this, not just for consistency but also for accessibility reasons.

Having seen your later work, I think this is a habit that you're now in, so keep up the good work!